components/logo.svg

```diff
@@ -1,7 +1,7 @@
 <svg viewBox="0 0 100 100" preserveAspectRatio="none" fill="none"
   xmlns="http://www.w3.org/2000/svg">
   <path d="M 85 15
-          L 15 85" style="strokeOpacity: 0.8" />
+          L 15 85" style="strokeOpacity: 0.8; animation-duration: 0.7071s" />
```

Comment:
not ideal, but can't figure out how to target this diagonal path via css modules

Reply:
ship it!!